src/averageAges.js

```diff
@@ -38,6 +47,24 @@ function calculateMenAverageAge(people, century) {
  */
 function calculateWomenAverageAge(people, withChildren) {
   // write code here
+  const womenWithChildren = people.filter(person => {
```

Comment:
Use the `Array.prototype.some()` method to simplify the `womenWithChildren` filtering logic.